javascript/advanced_examples/gdx_using_two_devices.html

```diff
@@ -126,15 +126,15 @@ <h1>Vernier Go Direct<sup>&reg</sup> JavaScript Advanced Example</h1>
       })(deviceObject));
       deviceObject.device.open(false);
         
-      } catch (err) {
-        console.error(err);
-        deviceObject.device = null;
-      }
-    };
-       
-      // configuration for the chart, change all initial chart settings here
-      var config = {
-			type: 'line',
+    } catch (err) {
+      console.error(err);
+      deviceObject.device = null;
+    }
+  };
+      
+  // configuration for the chart, change all initial chart settings here
+  var config = {
```

Comment:
We should be using either let or const depending on if this reference is redefined later. Looks like a const to me